src/olympia/blocklist/mlbf.py

```diff
@@ -21,15 +22,47 @@
 
 
 def ordered_diff_lists(
-    previous: List[str], current: List[str]
+    current: List[str],
+    previous: List[str],
 ) -> Tuple[List[str], List[str], int]:
-    current_set = set(current)
     previous_set = set(previous)
     # Use lists instead of sets to maintain order
-    extras = [x for x in current if x not in previous_set]
-    deletes = [x for x in previous if x not in current_set]
-    changed_count = len(extras) + len(deletes)
-    return extras, deletes, changed_count
+    return [x for x in current if x not in previous_set]
+
+
+def get_not_blocked_items(all_blocks_ids: List[int]) -> List[Tuple[str, str]]:
```

Comment:
strictly, this type definition isn't correct - the return is a type of queryset rather than `List`, unless the function is meant to cast the queryset into a `list(...)` before returning.